code/game/alternate_appearance.dm

```diff
@@ -71,6 +71,8 @@
 	var/list/alternate_appearances //the alternate appearances we own
 	var/list/viewing_alternate_appearances //the alternate appearances we're viewing, stored here to reestablish them after Logout()s
 	//these lists are built as necessary, so atoms aren't all lugging around empty lists
+	var/drag_start = 0
```

Comment:
Same here, please add some autodoc comments here so it's clear what these do.

Reply:
https://github.com/ParadiseSS13/Paradise/pull/21052/commits/d7b1c60523854173f455f567c3d40a4b44cf2902